critwall/src/main/java/org/opencadc/critwall/FileSyncJob.java

```diff
@@ -208,9 +210,8 @@ private void doSync() {
                     }
                     
                     // attempt to sync file
-                    log.debug("FileSyncJob.SYNC " + artifactLabel + " urls=" + urlList.size() + " attempts=" + retryCount);
+                    log.info("FileSyncJob.SYNC " + artifactLabel + " urls=" + urlList.size() + " attempts=" + retryCount);
```

Comment:
this message is supposed to be at debug level; the intent is to have 1 "start" and 1 "end" message and maybe we'll get rid of the "start" message in future; don't want any info messages in the loop.

Reply:
Okay. Changed back to debug level.